ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/radiobutton/v1/radiobutton/radiobutton.html

```diff
@@ -0,0 +1,33 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2022 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+<html>
+    <head></head>
+    <body>
+        <sly data-sly-use.renderer="${'radiobutton.js'}" data-sly-use.clientlib="${'/libs/granite/sightly/templates/clientlib.html'}" data-sly-use.label="${renderer.labelPath}" data-sly-use.shortdescription="${renderer.shortDescriptionPath}" data-sly-use.longdescription="${renderer.longDescriptionPath}" data-sly-use.errormessage="${renderer.errorMessagePath}" data-sly-use.questionmark="${renderer.questionMarkPath}" /> 
+        <div data-sly-use.formstructparser="com.adobe.cq.forms.core.components.models.form.FormStructureParser" data-sly-use.radiobutton="com.adobe.cq.forms.core.components.models.form.RadioButton" data-sly-use.component="com.adobe.cq.wcm.core.components.models.Component" class="cmp-adaptiveform-radiobutton" data-cmp-is="adaptiveFormRadioButton" id="${component.id}" data-cmp-adaptiveformcontainer-path="${formstructparser.formContainerPath}"> 
+            <div data-sly-call="${label.label @componentId=component.id, labelValue=radioButton.label.value, labelVisible=radioButton.label.visible, bemBlock='cmp-adaptiveform-radiobutton'}"></div> 
+            <div data-sly-call="${questionMark.questionMark @componentId=component.id, longDescription=radioButton.description, bemBlock='cmp-adaptiveform-radiobutton'}"></div> 
+            <div data-sly-list="${radioButton.enums}">
+                <input type="radio" name="${radioButton.name}" class="cmp-adaptiveform-radibButton__widget" readonly="${radioButton.readOnly}" required="${radioButton.required}" value="${item}">
+                <label>${radioButton.enumNames[itemList.index]}</label>
```

Comment:
`<label><input .../></label>` This allows selecting the radiobutton when label is clicked.
The second this is ensure the top label doesn't map to single radiobutton.

check with @gahuja1991 for checkbox group implementation

Reply:
done